site/content/en/docs/Installation/Creating Cluster/eks.md

```diff
@@ -41,6 +41,14 @@ For Agones to work correctly, we need to allow UDP traffic to pass through to ou
 * Select **Inbound Rules**
 * **Edit Rules** to add a new **Custom UDP Rule** with a 7000-8000 port range and an appropriate **Source** CIDR range (`0.0.0.0/0` allows all traffic)
 
+{{% feature publishVersion="1.44.0" %}}
```

Comment:
This feature was publish in 1.43, so we should remove this block so that it will get published right away.